Create the 'translatedabstract' environment

LucasHagen opened the pull request:
This pull request is a proposal/suggestion that improves three aspects of the _translated abstract_:

1. The title of the _translated abstract_ can now be defined together with the document's title, making it easier to change both when the title needs to be changed. This is achieved by adding a `\translatedtitle{...}` directive.
2. Same benefit from the previous item, but for keywords. The _normal_ keywords are defined in a different place from the translated ones, now they can be defined in the same place, making it easier for keyword changes. This creates a new directive to define the translated keywords: `\translatedkeyword{...}`
3. The `englishabstract` environment is not necessarily the abstract in English, it's actually the abstract in the _other_ language of the document. Now calling it `translatedabstract` makes it clearer and removes the necessity of any other parameters (translated title and keywords). **Warning: this only considers documents in Portuguese and English.**
4. Documents (using the `translatedabstract`) without a translated title or any translated keywords will receive a warning with instructions on how to use them.

This PR also updates the usages in the examples and keeps the original `englishabstract` environment as deprecated, making this PR a non-breaking change.

Extras:
- Updated keyword usage documentation, all keywords (first word) should start with uppercase letters. 

TODOs:
- Update `CHANGELOG`
- ~~Deprecate the `englishabstract` environment~~ (done)

LucasHagen commented:
Associated with [Issue #44](https://github.com/schnorr/infufrgs/issues/44).

schnorr commented:
Hi @LucasHagen , thanks for your pull request to organized this long-time issue. I liked your solution and look forward to merge. But before doing that, can you enrich your pull request with a deprecation notice in the `englishabstract` document? Thank you.

LucasHagen commented:
@schnorr , I've added a warning in the englishabstract. I've also updated the example for ORG-mode, but since I don't have any experience using it, could you check if those changes make sense?

schnorr commented:
Thanks @LucasHagen for the improvements, and sorry for the delay. Let's hope everyone accepts this modification that I believe is definitely a step forward. All examples have been updated so everyone should easily get acquainted. One of the org files is okay (`dis.org`) but the other one seems a bit old. I will keep it as is till we have more time to review or purge. Thanks again for your contribution!
